### ISSUE TITLE
false/null/true/number/string value is not supported

### ISSUE THREAD

**Issue by hanachin** [siman-man/jt#1]
These values are valid JSON-text
https://tools.ietf.org/html/rfc7159#section-2

```
% echo 'false' | jt
/usr/local/var/rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/jt-0.1.0/lib/jt.rb:27:in `parse': undefined method `empty?' for false:FalseClass (NoMethodError)
        from /usr/local/var/rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/jt-0.1.0/lib/jt.rb:17:in `initialize'
        from /usr/local/var/rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/jt-0.1.0/lib/jt.rb:12:in `new'
        from /usr/local/var/rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/jt-0.1.0/lib/jt.rb:12:in `create'
        from /usr/local/var/rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/jt-0.1.0/exe/jt:5:in `<top (required)>'
        from /usr/local/var/rbenv/versions/2.4.2/bin/jt:23:in `load'
        from /usr/local/var/rbenv/versions/2.4.2/bin/jt:23:in `<main>'
```
```
% echo 'null' | jt
/usr/local/var/rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/jt-0.1.0/lib/jt.rb:27:in `parse': undefined method `empty?' for nil:NilClass (NoMethodError)
        from /usr/local/var/rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/jt-0.1.0/lib/jt.rb:17:in `initialize'
        from /usr/local/var/rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/jt-0.1.0/lib/jt.rb:12:in `new'
        from /usr/local/var/rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/jt-0.1.0/lib/jt.rb:12:in `create'
        from /usr/local/var/rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/jt-0.1.0/exe/jt:5:in `<top (required)>'
        from /usr/local/var/rbenv/versions/2.4.2/bin/jt:23:in `load'
        from /usr/local/var/rbenv/versions/2.4.2/bin/jt:23:in `<main>'
```
```
% echo 'true' | jt
/usr/local/var/rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/jt-0.1.0/lib/jt.rb:27:in `parse': undefined method `empty?' for true:TrueClass (NoMethodError)
        from /usr/local/var/rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/jt-0.1.0/lib/jt.rb:17:in `initialize'
        from /usr/local/var/rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/jt-0.1.0/lib/jt.rb:12:in `new'
        from /usr/local/var/rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/jt-0.1.0/lib/jt.rb:12:in `create'
        from /usr/local/var/rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/jt-0.1.0/exe/jt:5:in `<top (required)>'
        from /usr/local/var/rbenv/versions/2.4.2/bin/jt:23:in `load'
        from /usr/local/var/rbenv/versions/2.4.2/bin/jt:23:in `<main>'
```
```
% echo '1' | jt
/usr/local/var/rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/jt-0.1.0/lib/jt.rb:27:in `parse': undefined method `empty?' for 1:Integer (NoMethodError)
        from /usr/local/var/rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/jt-0.1.0/lib/jt.rb:17:in `initialize'
        from /usr/local/var/rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/jt-0.1.0/lib/jt.rb:12:in `new'
        from /usr/local/var/rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/jt-0.1.0/lib/jt.rb:12:in `create'
        from /usr/local/var/rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/jt-0.1.0/exe/jt:5:in `<top (required)>'
        from /usr/local/var/rbenv/versions/2.4.2/bin/jt:23:in `load'
        from /usr/local/var/rbenv/versions/2.4.2/bin/jt:23:in `<main>'
```
```
% echo '"hi"' | jt
/usr/local/var/rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/jt-0.1.0/lib/jt.rb:28:in `parse': undefined method `shift' for "hi":String (NoMethodError)
        from /usr/local/var/rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/jt-0.1.0/lib/jt.rb:17:in `initialize'
        from /usr/local/var/rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/jt-0.1.0/lib/jt.rb:12:in `new'
        from /usr/local/var/rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/jt-0.1.0/lib/jt.rb:12:in `create'
        from /usr/local/var/rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/jt-0.1.0/exe/jt:5:in `<top (required)>'
        from /usr/local/var/rbenv/versions/2.4.2/bin/jt:23:in `load'
        from /usr/local/var/rbenv/versions/2.4.2/bin/jt:23:in `<main>'
```

**Comment by siman-man** [comment]
Thanks report! 😄

I fixed this issue with version 0.2.0 now! (maybe) 🔧

**Comment by hanachin** [comment]
I confirmed the fix, thank you 👍
